web/src/features/hfiCalculator/components/stationAdmin/LoggedInStatus.tsx

```diff
@@ -1,41 +1,42 @@
 import React from 'react'
-import makeStyles from '@mui/styles/makeStyles'
+import { styled } from '@mui/material/styles'
 import { isEmpty } from 'lodash'
 import { LockOutlined, LockOpenOutlined } from '@mui/icons-material'
 
+const PREFIX = 'LoggedInStatus'
+
+const Root = styled('div', {
+  name: `${PREFIX}-root`
+})({
+  display: 'flex',
+  alignItems: 'center'
+})
+
 export interface LoggedInStatusProps {
   isAuthenticated: boolean
   roles: string[]
   idir: string | undefined
 }
 
-const useStyles = makeStyles(() => ({
-  root: {
-    display: 'flex',
-    alignItems: 'center'
-  }
-}))
-
 const LoggedInStatus = ({ isAuthenticated, roles, idir }: LoggedInStatusProps) => {
-  const classes = useStyles()
   if (!isAuthenticated) {
     return <React.Fragment></React.Fragment>
```

Comment:
```suggestion
    return <div></div>
```

Reply:
I changed this to `<></>` which is shorthand for a React fragment: https://legacy.reactjs.org/docs/fragments.html#short-syntax. It allows us to use the `toBeEmptyDOMElement` in rtl.